python/foglamp/plugins/north/http_translator/http_translator.py

```diff
@@ -5,38 +5,28 @@
 # FOGLAMP_END
 
 """ HTTP Translator """
-
-import aiohttp
-import asyncio
 import json
 
-from foglamp import logger
-from foglamp.configuration_manager import ConfigurationManager
-from foglamp.translators.exceptions import *
+import requests
```

Comment:
this will be taken up in some future sprint as we are planning to use a common http send method in sending_process.py to be used both by omf translator and http translator.